src/Compilers/CSharp/Test/Emit/Emit/EndToEndTests.cs

```diff
@@ -100,10 +100,12 @@ void tryCompileDeepFluentCalls(int depth)
 
         [Fact]
         [WorkItem(33909, "https://github.com/dotnet/roslyn/issues/33909")]
+        [WorkItem(34880, "https://github.com/dotnet/roslyn/issues/34880")]
         public void DeeplyNestedGeneric()
         {
             int nestingLevel = (ExecutionConditionUtil.Architecture, ExecutionConditionUtil.Configuration) switch
             {
+                _ when ExecutionConditionUtil.IsMacOS => 100,
```

Comment:
This number seems pretty low even for debug. But I did find that 170 was too high.